<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CsgBooleanOptClass</class>
 <widget class="QWidget" name="CsgBooleanOptClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>290</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CsgBooleanOpt</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>请选择要进行布尔运算的两个模型：</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTextBrowser" name="selected_status"/>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="pushButton_and">
           <property name="text">
            <string>并集</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_differ">
           <property name="text">
            <string>差集</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_intersert">
           <property name="text">
            <string>交集</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
